client/handle_http.go

```diff
@@ -1120,12 +1140,6 @@ func walkDavDir(url *url.URL, namespace namespaces.Namespace, token string, dest
 func walkDirUpload(path string, client *gowebdav.Client, destPath string) ([]string, error) {
 	// List of files to return
 	var files []string
-	// Whenever this function is called, we should create a new dir on the server side for uploads
-	err := client.Mkdir(destPath+path, 0755)
```

Comment:
I was confused too but if you run a `object put` with a directory (with the -r flag) it seems to just make the directory for you. From the logs it seemed like it was redundantly making directories so that is why I removed it. let me know if that is just something weird on my end though but the tests seem to work for it.

Reply:
No, that's fine, I was just confused.